tests/test_page_inputs.py

```diff
@@ -178,7 +262,7 @@ def test_http_response_json():
     response = HttpResponse(url, body=b'{"key": "value"}')
     assert response.json() == {"key": "value"}
 
-    response = HttpResponse(url, '{"ключ": "значение"}'.encode("utf8"))
+    response = HttpResponse(url, body='{"ключ": "значение"}'.encode("utf8"))
```

Comment:
Do we have tests which check that positional body works?

Reply:
Good call! Tweaked some of the tests to test out the positional args alongside the keyword args. https://github.com/scrapinghub/web-poet/pull/22/commits/e7b4f081de979ce35d258629781d4342a40d1ac4